lib/iris/tests/results/integration/experimental/ugrid_save/TestBasicSave/ugrid_ex2_2d_triangular.cdl

```diff
@@ -0,0 +1,75 @@
+dimensions:
+	Mesh2_edge_N_faces = 2 ;
+	Mesh2_edge_N_nodes = 2 ;
+	Mesh2_face_N_edges = 3 ;
+	Mesh2_face_N_faces = 3 ;
+	Mesh2_face_N_nodes = 3 ;
```

Comment:
N.B. this use of separate dims for the last dim of different connectivity types is by design,
but it could seem wasteful.
We could change this, even create all "small" dimension values only once + re-use them